bin/compile

```diff
@@ -160,6 +160,11 @@ if [ "$NUM_PYTHON" = "2" ]; then
 
   # We remove the unneeded version of Python
   if [ "$DD_PYTHON_VERSION" = "2" ]; then
+    topic "*********************************** WARNING ************************************"
+    echo "Python 2 will be deprecated soon. Agent 7.x ships with Python 3 only." | indent
```

Comment:
Do we want to be more specific? ie "Python 2 will officially reach EOL on January 1st, 2020 and will no longer be supported."

Reply:
I want to avoid using a specific date that can change? but if we are ok, I can change it, yes

Reply:
Okay, that's reasonable.